x-pack/filebeat/module/mysqlenterprise/audit/test/mysql_audit_test.log-expected.json

```diff
@@ -0,0 +1,1100 @@
+[
+    {
+        "@timestamp": "2020-10-01T19:21:33.000Z",
+        "event.action": "mysql-startup",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "unknown",
+        "fileset.name": "audit",
+        "host.os.full": "x86_64-Linux",
+        "input.type": "log",
+        "log.offset": 0,
+        "mysqlenterprise.audit.class": "audit",
+        "mysqlenterprise.audit.connection_id": 0,
+        "mysqlenterprise.audit.id": 0,
+        "mysqlenterprise.audit.startup_data.mysql_version": "8.0.22-commercial",
+        "mysqlenterprise.audit.startup_data.server_id": 1,
+        "process.args": [
+            "/usr/sbin/mysqld"
+        ],
+        "related.user": [
+            "skip-grants user"
+        ],
+        "server.user.name": "skip-grants user",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:25:51.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-connect",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "success",
+        "event.type": [
+            "access",
+            "start"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 355,
+        "mysqlenterprise.audit.class": "connection",
+        "mysqlenterprise.audit.connection_data.connection_attributes._client_name": "libmysql",
+        "mysqlenterprise.audit.connection_data.connection_attributes._client_version": "8.0.22",
+        "mysqlenterprise.audit.connection_data.connection_attributes._os": "Linux",
```

Comment:
should these map to `host.os` fields?

Reply:
These should be in the `client` fields, `connection_data` is for client data and `startup_data` is for the host. However there was no `*.os` fields for `client` in the ECS definition :(